File: Python/car.py
from account import Account

class Car:
    id = int
    license = str
    driver = Account("","")
    passengers = int
    
    def __init__(self,license,driver) -> None:
        self.license = license
        self.driver = driver
        
    def printDataCar(self):
        print("Driver: {}".format(self.driver.name))
        print("License: {}".format(self.driver.license))
        print("Passengers: {}".format(self.passengers))

File: Java/Car.java
public class Car {
    Integer id;
    String license;
    String driver;
    Integer passenger;

    public void printDataCar(){
        System.out.println("Car license: "+this.license);
        System.out.println("Car driver: "+this.driver);
        System.out.println("Car passenger: "+this.passenger);
    }
    
}
